src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -156,19 +157,19 @@ const FormattedReport = ({ summaries, weekIndex, role }) => {
   };
 
   const bioSwitch = (userId, bioPosted) => {
-    const [isBioPosted, setIsBioPosted] = useState(bioPosted);
+    const [bioStatus, setBioStatus] = useState(bioPosted);
     return (
       <div>
         <div className="bio-toggle">
           <b>Bio announcement:</b>
         </div>
-        <div className="bio-toggle">
-          <ToggleSwitch
-            switchType="bio"
-            state={isBioPosted ? false : true}
-            handleUserProfile={() => {
-              handleChangeBioPosted(userId, isBioPosted);
-              setIsBioPosted(!isBioPosted);
+          <div className="bio-toggle">
+            <ToggleSwitch 
+              switchType="bio"
+              state={bioStatus}
+              handleUserProfile={(bio) => {
```

Comment:
```suggestion
        <div className="bio-toggle">
          <ToggleSwitch
            switchType="bio"
            state={bioStatus}
            handleUserProfile={(bio) => {
```
Extra indent and trailing whitespace.